CLDConfig/Overlay/Overlay.py

```diff
@@ -40,6 +40,7 @@
         "YokeBarrelCollection", "380",
         "YokeEndcapCollection", "380",
         "LumiCalCollection", "380"
+        "ECalBarrelEta", "380"
```

Comment:
I don't think LumiCalCollection can deal with an integration time of "380EcalBarrelEta".
```suggestion
        "LumiCalCollection", "380",
        "ECalBarrelEta", "380",
```